Rakefile

```diff
@@ -30,5 +30,35 @@ Rake::TestTask.new(:test) do |t|
   t.verbose = false
 end
 
+namespace :deploy do
+  desc "Update the sandbox app on Github"
+  task :update do
+    sandbox = "godmin-sandbox"
+    sandbox_dir = "/tmp/#{sandbox}-#{Time.now.strftime("%Y%m%d")}"
+    puts "Working in #{sandbox_dir}"
+    template_file = File.expand_path("../template.rb", __FILE__)
+    system("rm -rf #{sandbox_dir}")
+    system("mkdir #{sandbox_dir}")
+    system("cd #{sandbox_dir} && git clone git@github.com:varvet/#{sandbox}.git")
+    system("cd #{sandbox_dir}/#{sandbox} && rm -rf *")
+    system("cd #{sandbox_dir} && rails new #{sandbox} -m #{template_file} --without-engine")
```

Comment:
Yes, but I think it's more flexible to keep sqlite3 as development/test database and using postgresql for production (Heroku). Also, using postgresql as default adds the dependency of having postgresql installed since the `rake db:migrate` command otherwise will fail.